spec/requests/providers/statement_of_case_spec.rb

```diff
@@ -45,18 +46,18 @@
   end
 
   describe 'PATCH providers/proceedings_before_the_court' do
-    subject { patch providers_legal_aid_application_statement_of_case_path(legal_aid_application), params: params.merge(submit_button).merge(upload_button) }
     let(:entered_text) { Faker::Lorem.paragraph(3) }
-    let(:original_file) { uploaded_file('spec/fixtures/files/lorem_ipsum.pdf', 'application/pdf') }
-
-    let(:params) do
+    let(:original_file) { uploaded_file('spec/fixtures/files/documents/hello_world.pdf', 'application/pdf') }
+    let(:statement_of_case) { legal_aid_application.statement_of_case }
+    let(:params_statement_of_case) do
       {
-        statement_of_case: {
-          statement: entered_text,
-          original_files: [original_file]
-        }
+        statement: entered_text,
+        original_files: [original_file]
       }
     end
+    let(:params) { { statement_of_case: params_statement_of_case }.merge(submit_button).merge(upload_button) }
+
+    subject { patch providers_legal_aid_application_statement_of_case_path(legal_aid_application), params: params }
 
     context 'when the provider is authenticated' do
```

Comment:
I know you haven't touched this, but while you're in the spec ... this context is irrelevant. I would delete it.